packages/components/src/components/Passphrase/PassphraseTypeCardContent.tsx

```diff
@@ -0,0 +1,204 @@
+import { isAndroid } from '@trezor/env-utils';
+import { AnimatePresence, motion } from 'framer-motion';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { Card } from '../Card/Card';
+import { Columns } from '../Flex/Flex';
+import { PasswordStrengthIndicator } from './PasswordStrengthIndicator';
+import styled, { useTheme } from 'styled-components';
+import { Input } from '../form/Input/Input';
+import { spacingsPx, typography } from '@trezor/theme';
+import { useKeyPress } from '@trezor/react-utils';
+import { ChangeEvent } from 'react';
+import { Button } from '../buttons/Button/Button';
+import { WalletType } from './types';
+import { DOT } from './consts';
+import { Icon } from '../assets/Icon/Icon';
+import { motion as motionConfig } from '../../config';
+
+const PassphraseInput = styled(Input)`
+    input {
+        color: ${({ theme }) => theme.textSubdued};
+        ${typography.hint}
+    }
+`;
+
+const Actions = styled.div`
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+`;
+
+const ActionButton = styled(Button)`
+    margin-top: ${spacingsPx.xs};
+
+    &:first-child {
+        margin-top: 0;
+    }
+`;
+
+const Description = styled.div`
+    display: flex;
+    flex: 1;
+    color: ${({ theme }) => theme.textSubdued};
+    ${typography.label}
+`;
+
+type PassphraseTypeCardContentProps = {
+    submitLabel: React.ReactNode;
```

Comment:
I think (from what I saw in the code) we prefer import from react and not use the whole `React` reference.